sriovnet_switchdev_test.go

```diff
@@ -419,7 +419,7 @@ func TestGetVfRepresentorPortFlavour(t *testing.T) {
 		{netdev: "eth0", expected: PORT_FLAVOUR_PHYSICAL, shouldFail: false},
 		{netdev: "eth1", expected: PORT_FLAVOUR_PCI_PF, shouldFail: false},
 		{netdev: "eth2", expected: PORT_FLAVOUR_PCI_VF, shouldFail: false},
-		{netdev: "eth44", expected: PORT_FLAVOUR_UNKNOWN, shouldFail: false},
+		{netdev: "eth44", expected: PORT_FLAVOUR_PCI_SF, shouldFail: false},
```

Comment:
OK, so now we need a test case entry here to cover a case where we didnt match on typeToRegex map and actually hit line 296


so you need to add another `repContext` entry in `vfReps` with some unknown `PhysPortName`

Reply:
Done